Basic Movement

ShovelSquid opened the pull request:
Created a small website that allows player to control an image.
Able to move 4 ways with either arrow keys or WASD

ShovelSquid committed:
Basic Movement

Created a small website that allows player to control an image.
Able to move 4 ways with either arrow keys or WASD

rileyjohngibbs commented on branch.html:
```diff
@@ -0,0 +1,16 @@
+
+<body onkeydown = "KeyPresser(event)">
+	<div class="map">
+		<img id="sprite" src="file:///Users/IMG_3886.jpg">
```
Your file path points to an image file on the user’s machine. Needless to say, you can probably expect that file won’t be there in most cases! Better to add an image file to this project and point to that.

rileyjohngibbs reviewed (commented):
Great start. Make sure you know how all these values interact with each other, like the height and width of `.map` in the stylesheet, and why moving down adds to `top_` and moving up subtracts.

ShovelSquid committed:
Delete kaelen_reports.md

Not a Daily Reports Branch

ShovelSquid committed:
Map Increased

Map the map bigger, able to hold 41 x 41 50px tiles

ShovelSquid committed:
Merge branch 'Expiremental-Move-Website_1' of https://github.com/ShovelSquid/Time-Travel into Expiremental-Move-Website_1

ShovelSquid committed:
Added small list of lists

ShovelSquid committed:
Update branch.js

swerze committed:
Added tiles

woo

swerze committed:
Merge branch 'Expiremental-Move-Website_1' of https://github.com/ShovelSquid/Time-Travel into Expiremental-Move-Website_1

swerze committed:
Update branch.js

ShovelSquid committed:
Added tile dictionary

Also added css

swerze committed:
started create tile

swerze committed:
Merge branch 'Expiremental-Move-Website_1' of https://github.com/ShovelSquid/Time-Travel into Expiremental-Move-Website_1

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
FIXED ARRAY AND MADE IT BEAUTIFUL

THANK YOU RILEY

swerze committed:
More map work

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
Merge branch 'Expiremental-Move-Website_1' of https://github.com/ShovelSquid/Time-Travel into Expiremental-Move-Website_1

swerze committed:
AHA, numbers

swerze committed:
Update branch.js

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
Map editing

Added 100 divs for the map

ShovelSquid committed:
MAKING PUSH WOK

Its starting to work. yaay

ShovelSquid committed:
IT WORKS

ITS ALIVE

The map is functioning

ShovelSquid committed:
Added Tile Images

Now instead of Black and white representing the map, I got the correct images to display for the correct tiles.

ShovelSquid committed:
Update branch.js

ShovelSquid committed:
Position Data

Trying to get position of player
